models/labels/addresses/institution/identifier/cex/labels_cex_bitcoin.sql

```diff
@@ -17,3 +17,4 @@ SELECT blockchain
 , 'cex_' || blockchain AS model_name
 , 'identifier' AS label_type
 FROM {{ ref('cex_bitcoin_addresses') }}
+WHERE address IS NOT NULL
```

Comment:
You should drop the empty address instead of this, also any ones that are invalid (if it's not valid base58)

Reply:
I will add the filter in the next commit.